src/main/java/io/appform/dropwizard/actors/common/Constants.java

```diff
@@ -13,8 +13,10 @@ public class Constants {
 
     public static final String DEFAULT_CONSUMER_CONNECTION_NAME = "__defaultconsumer";
 
+    public static final String DEFAULT_SIDELINE_PROCESSOR_CONNECTION_NAME = "__defaultsidelineprocessor";
+
     public static final Set<String> DEFAULT_CONNECTIONS = Sets.newHashSet(DEFAULT_PRODUCER_CONNECTION_NAME,
-            DEFAULT_CONSUMER_CONNECTION_NAME);
+            DEFAULT_CONSUMER_CONNECTION_NAME, DEFAULT_SIDELINE_PROCESSOR_CONNECTION_NAME);
```

Comment:
Separate consumer connection is not required. Will this be used for publishing ?